src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -95,6 +96,13 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
                 .build();
         }
         if ("DEFENDANT_RESPONSE_SPEC".equals(callbackParams.getRequest().getEventId())) {
+
+            if (caseData.getRespondToClaim().getHowMuchWasPaid() != null && caseData.getRespondToClaim().getHowMuchWasPaid().compareTo(caseData.getTotalClaimAmount()) < 0){
+                caseData = caseData.toBuilder().respondent1ClaimResponsePaymentAdmissionForSpec(RespondentResponseTypeSpec.PART_ADMISSION).build();
```

Comment:
caseData.toBuilder().respondent1ClaimResponsePaymentAdmissionForSpec(RespondentResponseTypeSpec.PART_ADMISSION).build();

define a Boolean instead to compare with